src/gui/properties/piecesbar.cpp

```diff
@@ -286,7 +286,10 @@ void PiecesBar::showToolTip(const QHelpEvent *e)
     }
     else {
         stream << simpleToolTipText();
-        stream << '\n' << tr("Hold Shift key for detailed information");
+        if (showDetailedInformation) // metadata are not available at this point
```

Comment:
But we checked `if(m_torrent->hasMetadata())` for `then` branch. Here, in `else`, it doesn't have them.